server/core/store_test.go

```diff
@@ -183,10 +184,15 @@ func TestLowSpaceScoreV2(t *testing.T) {
 		// store1's capacity is less than store2's capacity, but store2 has more available space,
 		bigger: NewStoreInfoWithAvailable(1, 2*units.GiB, 100*units.GiB, 3),
 		small:  NewStoreInfoWithAvailable(2, 100*units.GiB, 10*1000*units.GiB, 3),
+	}, {
+		// store2 has extra file size (70GB), it can balance region from store1 to store2 .
+		small:  NewStoreInfoWithDisk(1, 400*units.MiB, 6930*units.GiB, 7000*units.GiB, 400),
+		bigger: NewStoreInfoWithAvailable(2, 1500*units.GiB, 7000*units.GiB, 1.32),
```

Comment:
How about adding the case mentioned in the issue?

Reply:
fixed